internal/operators/manager.go

```diff
@@ -203,6 +245,14 @@ func (mgr *Manager) AnyOLMOperatorEnabled(cluster *common.Cluster) bool {
 func (mgr *Manager) ValidateHost(ctx context.Context, cluster *common.Cluster, host *models.Host) ([]api.ValidationResult, error) {
 	results := make([]api.ValidationResult, 0, len(mgr.olmOperators))
 
+	if hasMCEWithStorage(cluster.Cluster.MonitoredOperators) {
+		storageOperator, err := mgr.getStorageOperator(&cluster.Cluster)
```

Comment:
Sorry about silly questions: 
What if both ODF and LVM is enabled? Would it matter if only ODF has the minimum disk requirement? Is this because if both are enabled, we're definitely expected to use ODF?

Reply:
Not a silly question, infact you have just spotted a bug :)
Both enabled and one not fulfilling the minimum disk requirements would block the installation, however we can only bump disk requirements on the one that it's going to be used (this is the bug, at the moment we are bumping both storage operator's disk requirements if MCE is selected).
The decision of picking ODF over LVM is arbitrary, but I think we should define it to avoid undefined behaviour.

Reply:
Nevermind, I was wrong, we do bump only the correct operator's disk requirements

Reply:
Ah that makes sense! Thank you for the explanation!!

> The decision of picking ODF over LVM is arbitrary, but I think we should define it to avoid undefined behaviour.
Agreed! Maybe doc it in code and in the user-facing docs? I think we have a doc folder for operators